ui/page/governance/governance_page.go

```diff
@@ -98,68 +84,58 @@ func (pg *Page) HandleUserInteractions() {
 	}
 
 	for pg.enableGovernanceBtn.Clicked() {
-		go pg.consensusPage.FetchAgendas()
 		go pg.WL.MultiWallet.Politeia.Sync()
-		pg.proposalsPage.isSyncing = pg.multiWallet.Politeia.IsSyncing()
+		pg.Display(NewProposalsPage(pg.Load))
```

Comment:
`pg.proposalsPage.isSyncing = pg.multiWallet.Politeia.IsSyncing()` is done when the proposals page is displayed by `pg.Display(NewProposalsPage(pg.Load))`.

`pg.consensusPage.FetchAgendas()` is called whenever the user clicks on that tab and the Consensus Page is displayed.